crates/subspace-networking/src/bin/subspace-bootstrap-node/main.rs

```diff
@@ -15,14 +15,17 @@ use subspace_networking::libp2p::multiaddr::Protocol;
 use subspace_networking::{
     peer_id, BootstrappedNetworkingParameters, Config, MemoryProviderStorage,
     NetworkingParametersManager, ParityDbProviderStorage,
-    SWARM_PENDING_TO_ESTABLISHED_CONNECTIONS_FACTOR,
 };
 use tracing::info;
 
-// The default maximum incoming connections number for the peer.
+/// The default maximum established incoming connections number for the peer.
 const MAX_ESTABLISHED_INCOMING_CONNECTIONS: u32 = 300;
-// The default maximum outgoing connections number for the peer.
+/// The default maximum established outgoing connections number for the peer.
 const MAX_ESTABLISHED_OUTGOING_CONNECTIONS: u32 = 300;
+/// The default maximum pending incoming connections number for the peer.
+const MAX_PENDING_INCOMING_CONNECTIONS: u32 = 900;
+/// The default maximum pending outgoing connections number for the peer.
+const MAX_PENDING_OUTGOING_CONNECTIONS: u32 = 900;
```

Comment:
Let's set these as default values for CLI parameters rather than having parameters as `Option<u32>` and then handling default value in the code. It helps with documentation since auto-generated help message in CLI will already show default values.